client/js/screens/BeginChallenge/styles.js

```diff
@@ -2,7 +2,61 @@ import { StyleSheet } from "react-native";
 import { colors, fonts } from "../../config/styles";
 
 const styles = StyleSheet.create({
- 
+  center: {
+    alignItems: "center",
+    flex: 1.25,
+    justifyContent: "space-around",
+    paddingTop: 120
+  },
+  flexText: {
+    alignItems: "center",
+    flex: 0.9,
+    justifyContent: "space-between",
+    paddingBottom: 60,
+    paddingTop: 30
+  },
+  background: {
+    width: "100%",
+    height: "100%"
+  },
+  text: {
+    fontFamily: fonts.light,
+    textAlign: "center",
+    fontSize: 14,
+    width: 300
+  },
+  logo: {
+    width: 180,
+    height: 30,
+    paddingBottom: 0
```

Comment:
use flexbox resizemode for your images instead of giving fixed height and width.

Reply:
Resolved